source/.htaccess.haml

```diff
@@ -414,6 +414,7 @@ directory_index: false
   Redirect 301 /case-studies/osas-test-lab /user-stories
   Redirect 301 /case-studies /user-stories
   Redirect 301 /rdo-manager /tripleo
+  Redirect 301 /documentation/packaging /documentation/rdo-packaging
```

Comment:
Can we also add Redirect for old URL /packaging/ to redirect here?
I got reports some folks had it bookmarked in the past and now it's 404.